msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -49,34 +50,73 @@ With the pre-meeting app experience, you can find and add meeting apps and perfo
-    > * If the in-meeting dialog box is rendered successfully, you are notified that the results successfully downloaded.
-    > * Your app manifest specifies the places where you want them to appear. The context field is used for this purpose. It can also be part of a share-tray experience, subject to specified design guidelines.
+# [Mobile](#tab/mobile)
+
+After entering the meeting and adding the app as per steps to use tabs during a meeting from desktop or web, the app is visible in mobile Teams meeting under the **Apps** section. Selecting **Apps** will show the list of apps and user can launch any of the app as an in meeting side panel of the app. User can interact with the app in it and launch it for users.
 
-    The following image illustrates the in-meeting side panel:
+The in-meeting dialog box is displayed where you can enter your response as a feedback.
 
-    ![In-meeting side panel](../assets/images/apps-in-meetings/in-meeting-dialog.png)
+![Mobile dialog box view](../assets/images/apps-in-meetings/mobile-in-meeting-dialog-view.png)
+
+> [!NOTE]
+> You do not require to change the app manifest for the apps to work on mobile.
+
+---
+
+> [!NOTE]
+> * Apps can leverage the Teams Client SDK to access the `meetingId`, `userMri`, and `frameContext` to render the experience appropriately.
+> * If the in-meeting dialog box is rendered successfully, you are notified that the results successfully downloaded.
+> * Your app manifest specifies the places where you want them to appear. The context field is used for this purpose. It can also be part of a share-tray experience, subject to specified design guidelines.
+
+The following image illustrates the in-meeting side panel:
+
+![In-meeting side panel](../assets/images/apps-in-meetings/in-meeting-dialog.png)
+
+The following table describes the behavior of app when it is approved and not approved:
+
+|App capability |Behavior if app is approved | Behavior if app is not approved |
+|---|---|---|
+| Meeting extensibility | The app will be approved and will start appearing in the meetings. | The app will not appear in the meeting stage of the mobile client. |
 
 #### Post-meeting app experience
 
 With post-meeting app experience, you can view the results of the meeting such as poll survey results or feedback. Select <img src="~/assets/images/apps-in-meetings/plusbutton.png" alt="Plus button" width="30"/> to add a tab and get meeting notes and results on which organizers and attendees must take action.
 
 The following image displays the **Contoso** tab with results of poll and feedback received from meeting attendees:
 
+# [Desktop](#tab/desktop)
+
 ![Post meeting view](../assets/images/apps-in-meetings/PostMeeting.png)
 
+# [Mobile](#tab/mobile)
+
+![Mobile post meeting view](../assets/images/apps-in-meetings/mobilePostMeeting.png)
+
+---
+
 > [!NOTE]
 > Tab layout must be organized when there are more than ten polls or surveys.
```

Comment:
ten---> mention 10 as per MSTP "use numerals for 10 or greater. "

Reply:
Its the other way round we use numbers for more than 10. less than 10 we call out the numbers